__tests__/uniqId.spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import MyBlock from 'b:MyBlock';
+
+it('Should generate id', () => {
+    expect(shallow(<MyBlock/>).prop('id'))
+        .toMatch(/uniq\$/);
```

Comment:
I suggest generation of two nodes inside content (i.e. for input/label) and check equality of attributes (which are based on `generateId()` from same context)